book.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'rental'
+class Book
+  attr_accessor :author, :title, :rentals
+
+  def initialize(title, author, _rentals)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    @rentals << Rental.new(date, self, person)
+  end
```

Comment:
- [ ] Now we have to modify this method as requested by the reviewer. There is no need for `@rentals <<` since creating a Rental object already adds the rentals to the array.

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```